soak/helm/ack-soak-test/values.yaml

```diff
@@ -0,0 +1,13 @@
+# Default values for ack-soak-test.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+serviceName: "" # provide during helm install command
```

Comment:
I initially understand this as K8s idea of "service" rather than the AWS service name. Perhaps this could be clarified as "awsService"?